projects/wmf_default.yaml

```diff
@@ -18,12 +18,12 @@ paths:
                   allows us to contact you quickly. Email addresses or URLs
                   of contact pages work well.
 
-                By using this API, you agree to Wikimedia's 
+                By using this API, you agree to Wikimedia's
```

Comment:
i know it's bad to have trailing spaces, but this change is entirely unrelated to the purpose of this commit/PR. could you please revert this?

We try to avoid stashing unrelated changes into pull requests, it convolutes the history and `git blame` becomes useless.